src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -104,24 +100,15 @@ private static TextSpan CreateSpan(SyntaxTokenList startOpt, SyntaxNodeOrToken s
         }
 
         private static int GetEndPosition(SyntaxNodeOrToken nodeOrToken)
-        {
-            if (nodeOrToken.IsToken)
-            {
-                return nodeOrToken.Span.End;
-            }
-            else
-            {
-                return nodeOrToken.AsNode().GetLastToken().Span.End;
-            }
-        }
+            => nodeOrToken.AsNode(out var node) ? node.GetLastToken().Span.End : nodeOrToken.Span.End;
 
         internal static void GetBreakpointSpan(VariableDeclaratorSyntax declaratorSyntax, out SyntaxNode node, out TextSpan? part)
         {
-            var declarationSyntax = (VariableDeclarationSyntax)declaratorSyntax.Parent;
+            var declarationSyntax = (VariableDeclarationSyntax)declaratorSyntax.Parent!;
```

Comment:
ecan we use `RoslynDebug.AssertNotNull` on the parent once so we don't need to keep suppressing nullable warnings?